AutomatedTesting/Gem/PythonTests/scripting/NodePalette_HappyPath_CanSelectNode.py

```diff
@@ -5,18 +5,20 @@
 SPDX-License-Identifier: Apache-2.0 OR MIT
 """
 
+from PySide2 import QtWidgets
+from editor_python_test_tools.utils import Report
+import azlmbr.legacy.general as general
+import editor_python_test_tools.pyside_utils as pyside_utils
```

Comment:
understood. I moved it since the other files I was using as a reference had the imports at the top. Also, it seems to be standard in many other languages to put imports and such at the top. I just wanted the script test behavior to be clearer as well.

Reply:
Python supports scoped imports (as does C#, Java, and many other languages). What this means is that you can import something in that brings it into scope for the entire file (module), a class within the file, or even a function. The benefit here to limit scope to the class/function is that during a module import, the scope of your tests will not have a chance to unexpected pollute the environment (this can become a real concern with parallelized execution and batched execution of our tests on Jenkins). 

Since it looks like the standard that the O3DE automation is trying to move towards is limiting the scope of imports to the test class level scope of the test file scope, we should make sure that this test fits that paradigm instead of using standards from other languages.

Reply:
moved the imports back down into the function underneath the preconditions comment.